sdgym/datasets.py

```diff
@@ -46,7 +46,7 @@ def _get_dataset_path(modality, dataset, datasets_path, bucket=None, aws_key=Non
     if dataset_path.exists():
         return dataset_path
 
-    local_path = Path(bucket) / dataset
+    local_path = Path(bucket) / dataset if bucket else Path(dataset)
```

Comment:
does this work on all operating systems or should we us `os.join`?

Reply:
I'm not sure, since I haven't tested it out for Windows, but I see your concern. I'll just switch it to using `os.path.join`.

Reply:
Actually it seems like using path operators should be compatible with all platforms since it creates a new `Path` object (not just a string) -https://docs.python.org/3/library/pathlib.html#operators